<launch>
  <arg name="INPUT" default="/kinect_head_remote/depth_registered/points" />
  <arg name="object_name" default="object" />

  <node name="service_save_pointcloud_to_pcd" type="nodelet" pkg="nodelet"
        args="standalone jsk_pcl_utils/PointCloudToPCD" output="screen"
        clear_params="true">
    <remap from="~input" to="$(arg INPUT)" />
    <rosparam subst_value="true">
      prefix: $(find jsk_2023_09_cook_from_recipe)/datas/saved_pcds/service_saved_$(arg object_name)
      duration: 0.0
      binary: true
      compressed: true
    </rosparam>
  </node>
</launch>
